fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,35 @@
+#![no_main]
+
+use std::io::Cursor;
+
+use lepton_jpeg::{decode_lepton, encode_lepton_feat, EnabledFeatures};
+
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: &[u8]| {
+    let r;
+
+    let mut output = Vec::new();
+
+    {
+        let mut writer = Cursor::new(&mut output);
+
+        // kepp the jpeg dimensions small otherwise the fuzzer gets really slow
+        let features = EnabledFeatures {
+            progressive: true,
+            max_jpeg_height: 503,
```

Comment:
isn't this too small ?
I would expect something several megapixels.

Reply:
it does 1000s of iterations a second. i'll add a comment